util/kubernetes/client/client.go

```diff
@@ -23,7 +23,7 @@ var (
 	// ErrReadNamespace is returned when the names could not be read from service account
 	ErrReadNamespace = errors.New("Could not read namespace from service account secret")
 	// DefaultImage is default micro image
-	DefaultImage = "micro/go-micro"
+	DefaultImage = "micro/cells:go"
```

Comment:
We should leave this as micro/go-micro and either pass the appropriate commands or specify micro/cells:go when we actually do `micro run` and pass image to the runtime. I don't think go-micro should reference this image.

Reply:
Overwriting it in `micro run` should be simple enough assuming it's correctly passed down the chain.